src/contentscript/overlay/root/pages/DappletsInfo/index.tsx

```diff
@@ -62,11 +61,11 @@ export const DappletsMainInfo: FC<DappletsMainInfoProps> = (props) => {
   const [visible, setVisible] = useState(false)
   const nodeBtn = useRef<HTMLButtonElement>()
   const nodeInput = useRef<HTMLInputElement>()
-  const [visibleContextId, setVisibleContextId] = useState({
-    visibleContext: [],
-  })
+  const messagesContainer = useRef<HTMLDivElement>()
+
   const [contextDeleteNone, setContextDeleteNone] = useState(false)
   const [addDisabled, setAddDisabled] = useState(false)
+  const [contextId, setContextId] = useState(null)
```

Comment:
contextId => contextIds, setContextId => setContextIds because an array of context ids is there.